CONTRIBUTING.md

```diff
@@ -300,6 +300,11 @@ Use your real name (sorry, no pseudonyms or anonymous contributions.)
 If you set your `user.name` and `user.email` git configs, you can sign your
 commit automatically with `git commit -s`.
 
+You can enable local [prepare-commit-msg](hack/git/hooks/prepare-commit-msg) hook to do this automatically:
+```
```

Comment:
nit: if you add a `bash` type-hint, GitHub will apply code-highlighting to the example below;

```suggestion
```bash
```